docs/themes/thxvscode/layouts/partials/footer.html

```diff
@@ -28,6 +28,8 @@
                 <ul class="links">
                     <li><a id="footer-privacy-link" href="https://privacy.microsoft.com/privacystatement"
                             target="_blank">Privacy</a></li>
+                    <li><a id="footer-consumer-health-privacy-link" href="https://privacy.microsoft.com/en-US/microsoft-consumer-health-data-privacy-policy"
```

Comment:
I think you can omit the en-us from the url, and let the site redirect to the right language based on browser prefs.

Reply:
I've updated the URL to match the one referenced in the ADO ticket.